src/_includes/feature_lists/self-hosted.njk

```diff
@@ -13,11 +42,40 @@
             values: ['5', 'Unlimited'],
             info: "<p>FlowFuse allows multiple team members to collaborate on the same Node-RED flows.</p>"
         }, {
-            id: "nodered-instances",
-            label: "Number of Node-RED instances",
-            values: ['5', 'Licensed amount'],
-            info: "<p>Number of Node-RED runtimes, either as hosted instances or as devices</p>"
+            id: "team-library",
+            label: "Team Library",
+            values: [null, 'check'],
+            info: "<p>Setup standard nodes and flows that can be shared with all team members.</p>"
+        }, {
+            id: "dashboard",
+            label: "Node-RED Dashboard",
+            values: ['check', 'check'],
+            info: "<p>Visualize your data with the powerful and intuitive <a href=\"https://dashboard.flowfuse.com\" target=\"blank\">Node-RED Dashboard</a></p>"
+        }, {
+            id: "multi-user-dashboard",
+            label: "Personalised Multi-user Dashboards",
+            values: [null, 'check'],
+            info: "<p>Personalised Multi-user Dashboard allow you to build applications that provide unique data to each user <a href=\"https://flowfuse.com/blog/2024/01/dashboard-2-multi-user\" target=\"blank\">Learn more</a></p>"
         }, {
+            id: "pipelines",
+            label: "DevOps Pipelines",
+            values: [null, 'check'],
+            info: "<p>DevOps pipelines allows for you to setup different environments for development, testing and production Node-RED instances.</p>"
+        }, {
+            id: "project-nodes",
+            label: "Seamless Data Stream",
+            values: [null, 'check'],
+            info: "<p><a href=\"https://github.com/FlowFuse/nr-project-nodes\" target=\"blank\">FlowFuse Project Nodes</a> enable the passing of data and messages between your Node-RED projects, included with all FlowFuse Projects by default.</p>"
+        }, {
+            id: "blueprints",
+            label: "Access to Blueprints",
+            values: [null, 'check'],
+            info: "<p><a href=\"https://flowfuse.com/blog/2023/10/blueprints/\" target=\"blank\">FlowFuse's Blueprints</a> simplify Node-RED deployments by offering pre-built, customizable flows for specific use cases.</p>"
+        }]
+    }, {
+        label: "Security & Compliance",
+        hosting: "self-hosted",
+        rows: [{
             id: "nr-hosting",
             label: "Node-RED Hosting",
```

Comment:
Either we should remove this all together, or put it the No-Ops section.

Reply:
I've moved it under No-Ops